mppsolar/libs/schedule.py

```diff
@@ -0,0 +1,174 @@
+from enum import StrEnum, auto
+from time import sleep, time
+import yaml
+import logging
+from mppsolar.sender import get_output
+
+log = logging.getLogger("schedule")
+
+class Schedule:
```

Comment:
I like this concept but I think this misses a few things, especially for the more complicated porttypes
A lot of the complexity you are removing has been added to cover these other porttypes, a lot of the reason behind powermon and the expanded yaml config is to cover the variation between different hardware / porttypes